packages/dante3/src/popovers/menuBar.tsx

```diff
@@ -171,7 +179,7 @@ export default function MenuBar({ editor, fixed }) {
 
           <DanteTooltipLink
             selected={editor.isActive("link")}
-            editor={editor}
+            //editor={editor}
```

Comment:
let's keep this from now

Reply:
Alright. I'll add the editor into the tooltip props instead.